.github/workflows/build.yml

```diff
@@ -44,11 +44,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Node 14
-        uses: actions/setup-node@v2
+      - name: Install Node.js
+        uses: actions/setup-node@v3
         with:
-          node-version: '14'
+          node-version: '18.x'
       - name: Install npm packages
-        run: npm install -g sass postcss-cli postcss autoprefixer
+        run: npm install
```

Comment:
You may want to change this to `npm ci`

Reply:
@hieplpvip done in https://github.com/VNOI-Admin/OJ/pull/324/commits/3d7f306a319001e87325907450907bdb2667b628. I'm not really familar with `npm` as I usually use `pnpm`, so yeah.